var/spack/repos/builtin/packages/damaris/package.py

```diff
@@ -44,13 +46,16 @@ class Damaris(CMakePackage):
         default=False,
         description="Enables building of Python enabled Damaris library using Boost::python",
     )
+    extends("python", when="+python")
 
     depends_on("xsd")
     depends_on("xerces-c")
     depends_on("mpi")
     depends_on("cmake@3.18.0:", type=("build"))
-    depends_on("boost+thread+log+filesystem+date_time" "@1.67:")
-    depends_on("boost+thread+log+filesystem+date_time+python+numpy" "@1.67:", when="+python")
+    depends_on("boost+thread+log+filesystem+date_time+system" "@1.67:")
+    depends_on(
+        "boost+thread+log+filesystem+date_time+system+python+numpy" "@1.67:", when="+python"
+    )
```

Comment:
```suggestion
    depends_on("boost+python", when="+python")
```
These constraints get merged during the solve anyway